sorts/odd_even_transposition_parallel.py

```diff
@@ -92,39 +99,60 @@ def odd_even_transposition(arr):
     >>> odd_even_transposition(unsorted_list) == sorted(unsorted_list + [1])
     False
     """
+    # spawn method is considered safer than fork
+    multiprocessing_context = mp.get_context("spawn")
+
     process_array_ = []
     result_pipe = []
     # initialize the list of pipes where the values will be retrieved
     for _ in arr:
-        result_pipe.append(Pipe())
+        result_pipe.append(multiprocessing_context.Pipe())
     # creates the processes
     # the first and last process only have one neighbor so they are made outside
     # of the loop
-    temp_rs = Pipe()
-    temp_rr = Pipe()
+    temp_rs = multiprocessing_context.Pipe()
+    temp_rr = multiprocessing_context.Pipe()
     process_array_.append(
-        Process(
+        multiprocessing_context.Process(
```

Comment:
I add a commit to access the multiprocessing context explicitly. The context instance is set to `spawn` above separating from the global default context.

```python
import multiprocessing as mp

mp.set_start_method("spawn")  # set global default context

process = mp.Process(...)
pipe = mp.Pipe(...)
lock = mp.Lock(...)
```

vs.

```python
import multiprocessing as mp

ctx = mp.get_context("spawn")

# Use the spawn context explicitly
process = ctx.Process(...)
pipe = ctx.Pipe(...)
lock = ctx.Lock(...)
```

If this change is not desired or makes the code too complex, I can remove the last commit in the PR. cc @cclauss